src/mount/special_inode.cc

```diff
@@ -41,9 +41,14 @@ const Attributes InodeStats::attr =
 	  {{'f', 0x01,0xA4, 0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,1, 0,0,0,0,0,0,0,0}};
 const Inode InodeStats::inode_ = SPECIAL_INODE_STATS;
 
-// 0x0100 == 0b100000000 == 0400
+// Win: 0x0124 == 0b100100100 == 0444
+// Other OSs: 0x0100 == 0b100000000 == 0400
```

Comment:
By Other OS's you mean Unix systems?

Reply:
Well, basicly yes. I was referring to other than Windows OSs.

Reply:
I guess this is more of a nit than anything (so I don't care), but there are more operating systems than Unix systems like Linux or macOS.

https://eylenburg.github.io/pics/Eylenburg_Operating_System_Timeline_Family_Tree.svg